Jobs.Fetcher.Twitter/Jobs/Ads/AdsAccountsQuery.cs

```diff
@@ -40,10 +40,14 @@ public class AdsAccountsQuery : AbstractTwitterFetcher {
             }
 
             void ProccessAdsAccountResult(ITwitterRequestIterator<AdsAccountsResponse, string> iterator) {
-
-                while (!iterator.Completed) {
-                    var adsAccountPage = iterator.NextPageAsync().GetAwaiter().GetResult();
-                    DbWriter.WriteAdsAccounts(user.Id, username, adsAccountPage.Content, adsContext, GetLogger());
+                try {
+                    while (!iterator.Completed) {
+                        var adsAccountPage = iterator.NextPageAsync().GetAwaiter().GetResult();
+                        DbWriter.WriteAdsAccounts(user.Id, username, adsAccountPage.Content, adsContext, GetLogger());
+                    }
+                }catch (Exception e) {
+                    GetLogger().Error($"Could not fetch Twitter Video Libraries for {username}");
```

Comment:
I might be getting something wrong, but do you need GetLogger here? I always assumed the existence of `Logger` in the `AbstractJob` class. If this is correct, you could also change it for the function call `DbWriter.WriteAdsAccounts` above